tests/test_recipient_csv.py

```diff
@@ -937,6 +937,28 @@ def test_multiple_sms_recipient_columns(international_sms):
     assert recipients.has_errors
 
 
+@pytest.mark.parametrize('international_sms', (True, False))
```

Comment:
I don't think international_sms adds anything to this test.
It would more interesting to know that the test passes if `phone number` has spaces, underscores, no spaces, etc.

Reply:
Thanks, that's a good suggestion, I tested for those cases and discovered some weirdness :D

Reply:
I added a comment to the test explaining the oddity. This is pre-existing behaviour, but we probably didn't know about it. It shouldn't affect our users.